batch/create-batch-tables.sql

```diff
@@ -275,6 +289,10 @@ BEGIN
 
   SELECT state INTO cur_state FROM instances WHERE name = in_instance_name;
 
+  UPDATE instances
+  SET time_deactivated = in_timestamp
+  WHERE name = in_instance_name;
+
```

Comment:
Need to account for the places where instance can be assumed to be NULL or is set to NULL in the attempts table.